### PR TITLE
pkg/checkpoint: support both CRI v1alpha1 and v1alpha2

### PR DESCRIPTION
**Pull request by ericchiang**
Updates https://github.com/kubernetes-incubator/bootkube/issues/934

Generated proto files taken from the following paths:

https://github.com/kubernetes/kubernetes/tree/v1.9.6/pkg/kubelet/apis/cri
https://github.com/kubernetes/kubernetes/tree/v1.10.0-rc.1/pkg/kubelet/apis/cri

### CODE REVIEW

**Comment by rphillips** [comment]
could you add a README in the v1alpha1 directory where those files came from?

**Comment by ericchiang** [comment]
> could you add a README in the v1alpha1 directory where those files came from?

Yeah will do. Going to let CI see if this works first.

File: pkg/checkpoint/runtime_service.go [hunk 1]
```diff
@@ -93,53 +88,123 @@ func (r *remoteRuntimeService) localRunningPods() map[string]*v1.Pod {
...
+				},
+			},
+		})
+		if err != nil {
+			glog.Errorf("ListContainers with filter from runtime service failed: %v", err)
+			return nil, err
+		}
+		for _, c := range resp.Containers {
+			if c.Metadata == nil {
+				glog.V(4).Infof("Container does not have metadata: %+v", c)
```
Comment (colemickens) [inline pkg/checkpoint/runtime_service.go]: Why do we skip here? When does this happen?

Comment (ericchiang) [reply pkg/checkpoint/runtime_service.go]: Just copied from the previous behavior.

Comment (colemickens) [reply pkg/checkpoint/runtime_service.go]: Yeah, I saw the removed block too, was just curious. not a big deal.

**Comment by ericchiang** [comment]
Testing this locally with v1.10-rc.1

**Comment by ericchiang** [comment]
This appears to work. I was able to get a v1.10-rc.1 cluster running and the checkpointer behaves fine.

File: pkg/checkpoint/cri/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Vendored CRI APIs
+
+We copy generated CRI APIs for a couple reasons:
+* When CRI promotes a version, it sometimes deletes the old alpha version.
+* Prevents importing `k8s.io/kubernetes`.
```
[the remark this answers is not in the document]
Comment (ericchiang) [reply pkg/checkpoint/cri/README.md]: I was gonna do that in a follow up (or let someone else do it). It's a much bigger diff :)

**Pull request merged**